src/pages/demo/linkTeasers/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { drupalClient } from '@/utils/drupalClient'
+import { GetStaticPropsContext, GetStaticPropsResult } from 'next'
+import { DrupalParagraph } from 'next-drupal'
+import { DrupalJsonApiParams } from 'drupal-jsonapi-params'
+import Container from '@/components/container'
+import LinkTeaser from '@/components/paragraph/link_teaser'
+
+interface LinkTeaserPageProps {
+  linkTeasers: DrupalParagraph[]
+}
+
+const LinkTeaserPage = ({ linkTeasers }: LinkTeaserPageProps) => {
+  if (!linkTeasers) linkTeasers = []
+
+  return (
+    <>
+      <Container className="container">
+        {linkTeasers.map((paragraph) => (
+          <ul key={paragraph.id} className="usa-unstyled-list">
```

Comment:
Needed to add <ul> since this component is typically consumed by list_of_link_teasers and will not render exact without it.

Reply:
Would it be better to move the `ul` outside of the the map so it's only generating list items or is that not how this component works?

Reply:
Good catch! will do